tests/baselines/reference/checkJsFiles_skipDiagnostics.symbols

```diff
@@ -0,0 +1,38 @@
+=== tests/cases/compiler/a.js ===
+
+var x = 0;
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+
+/// @ts-suppress
+x();
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+/// @ts-suppress
+x();
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+/// @ts-suppress
+x(
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+    2,
+    3);
+
+
+
+// @ts-suppress
+// come comment
+// some other comment
+
+// @anohter
+
+x();
+>x : Symbol(x, Decl(a.js, 1, 3))
+
+
+
+// @ts-suppress: no call signature

```

Comment:
This is just an arbitrary message rather than something indicating a specific diagnostic to suppress, right?

Could be useful to have a way to give a specific one to hide (say, something matching `@ts-suppress\(T(\d+)\)`), in a similar way how linter comments have a way to give a specific rule to disable; but that could be a future enhancement.